app/models/katello/content_view.rb

```diff
@@ -1,7 +1,7 @@
 module Katello
   # rubocop:disable Metrics/ClassLength
   class ContentView < Katello::Model
-    audited :associations => [:repositories, :environments]
+    audited :associations => [:repositories, :environments, :filters], :except => [:name, :label, :description]
```

Comment:
Not auditing name, label or description changes on CV since these are not changes that require publish.